js/src/wp-seo-post-scraper.js

```diff
@@ -465,6 +537,27 @@ import { setMarkerStatus } from "./redux/actions/markerButtons";
 		if ( data.setRefresh ) {
 			data.setRefresh( app.refresh );
 		}
+
+		const snippetEditorData = {
+			title: postDataCollector.getSnippetTitle(),
+			slug: postDataCollector.getSnippetCite(),
```

Comment:
Why is this called Cite? I know you didn't name it in this PR, but I'm still wondering

Reply:
This is very very old. This was originally named Cite because the containing element was an HTML `<cite>`.